packages/blunderbuss/src/blunderbuss.ts

```diff
@@ -129,6 +133,13 @@ export function blunderbuss(app: Probot) {
       const byConfig = issue ? config.assign_issues_by : config.assign_prs_by;
       const issuePayload = issue || pullRequest;
 
+      // Acquire the lock.
+      const lock = new DatastoreLock('blunderbuss', lockTarget);
+      const lockResult = await lock.acquire();
+      if (!lockResult) {
+        throw new Error('Failed to acquire a lock');
```

Comment:
Can you include the issue's info in the error, so it's easier to pinpoint in the logs?

Reply:
Done (with the next commit)